Jenkinsfile

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
This file was not tested, I do not have a Jenkins available in my environment but I expect it's ok. 
The idea is to integrate the Github PR with Jenkins to validate we do not break the main branch. 
It automates validation.